galaxy_ng/app/urls.py

```diff
@@ -46,3 +48,7 @@
     ),
     path("healthz", views.health_view),
 ]
+
+if settings.get("SOCIAL_AUTH_KEYCLOAK_KEY"):
+    urlpatterns.append(url("", include("social_django.urls", namespace="social")))
+    urlpatterns.append(path("login/", lambda request: redirect("/login/keycloak/", permanent=True)))
```

Comment:
I'm not sure making this a permanent redirect is a good idea. If someone decides to stop using social auth, it's going to make things difficult.

Reply:
At the very least, it will make things harder for us when we're doing development work :)